test/electron-renderer/ui/LyricsViewer_spec.js

```diff
@@ -1,12 +1,14 @@
-/* eslint-disable no-unused-expressions */
+/* eslint-disable no-unused-expressions, no-duplicate-imports, import/no-duplicates */
```

Comment:
some of the tests are actually expecting HOC wrapper which is being exported by default and won't pass if we use pure component class. The new tests require the unwrapped component itself in order to call its methods directly. So, I had to turn off this rule for this file and import two things from one file.  The other option is not to turn off eslint but instead create separate file for HOC wrapper.